src/GattCharacteristic.js

```diff
@@ -33,16 +33,20 @@ class GattCharacteristic extends EventEmitter {
     return Buffer.from(payload)
   }
 
-  async writeValue (value, offset = 0) {
+  async writeValue (value, options = {}) {
```

Comment:
In order to avoid breaking changes, can you make this signature backwards compatible?

I think that you can rename `options` to `optionsOrOffset` and check the provided input. If the user provides a number we can assume that it is an offset, alternatively it is an options object.